admin/app/components/solidus_admin/orders/index/component.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::Orders::Index::Component < SolidusAdmin::BaseComponent
+  def initialize(
+    page:,
+    badge_component: component('ui/badge'),
+    table_component: component('ui/table'),
+    pagination_component: component('ui/table/pagination'),
+    button_component: component("ui/button")
+  )
+    @page = page
+
+    @badge_component = badge_component
+    @table_component = table_component
+    @button_component = button_component
+    @pagination_component = pagination_component
+  end
+
+  def title
+    Spree::Order.model_name.human.pluralize
+  end
+
+  def prev_page_link
+    @page.first? ? nil : solidus_admin.url_for(host: request.host, port: request.port, **request.params, page: @page.number - 1)
+  end
+
+  def next_page_link
+    @page.last? ? nil : solidus_admin.url_for(host: request.host, port: request.port, **request.params, page: @page.next_param)
+  end
+
+  def batch_actions
+    []
+  end
+
+  def filters
+    []
+  end
+
+  def columns
+    [
+      number_column,
+      date_column,
+      customer_column,
+      total_column,
+      items_column,
+      payment_column,
+      shipment_column,
+    ]
+  end
+
+  def number_column
```

Comment:
Looking at the design files, it seems that the order number needs to be bolded unless the order is paid.

Reply:
Thank you for catching this! 🙏 
After revisiting the design files, I realized that:
- The order number should indeed be bolded if the order hasn't been paid or shipped.
- When the order is paid and shipped, the row will have a "fading effect" like the header: I've made some adjustments with the introduction of a `fade_row_proc` to ensure this behavior.

Let me know WDYT 👍